js/views/attachment_view.js

```diff
@@ -15,8 +15,13 @@
 
   var ImageView = Backbone.View.extend({
       tagName: 'img',
-      initialize: function(dataUrl) {
-          this.dataUrl = dataUrl;
+      initialize: function(objectUrl, contentType, blob) {
+        loadImage(blob, autoOrientedCanvas => {
```

Comment:
I think it might be easier to follow if the callback here was another method of the class, with a `.bind(this)`

Reply:
Good point! I pushed some new changes where I extracted `autoOrientImage` function that exposes a `Promise` based interface to `loadImage`.